config/database.yml

```diff
@@ -19,18 +19,18 @@ default: &default
   encoding: unicode
   # For details on connection pooling, see Rails configuration guide
   # http://guides.rubyonrails.org/configuring.html#database-pooling
-  pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
+  pool: <%= ENV.fetch("DATABASE_POOL_SIZE") { ENV.fetch("RAILS_MAX_THREADS", 5).to_i } %>
   connect_timeout: 2
   checkout_timeout: 2
   variables:
-      statement_timeout: <%= ENV.fetch('STATEMENT_TIMEOUT', 2500).to_i %>
+      statement_timeout: <%= ENV.fetch("STATEMENT_TIMEOUT", 2500).to_i %>
 
 development:
   <<: *default
-  url: <%= ENV.fetch('DATABASE_URL', '') %>
-  database: PracticalDeveloper_development
+  url: <%= ENV.fetch("DATABASE_URL", "postgres:///Forem_development") %>
```

Comment:
quick question: how come the backup to the ENV fetch here doesn't have localhost and port, as supposed to `export DATABASE_URL="postgresql://localhost:5432/$DATABASE_NAME`?

Reply:
I think because `localhost:5432` is the default:

```shell
$ psql postgres:///PracticalDeveloper_development
psql (13.1)
Type "help" for help.

PracticalDeveloper_development=#
```